examples/drivenet/drivenet_open_loop.ipynb

```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DriveNet Open Loop Evaluation\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./drivenet_train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "### What is open loop evaluation?\n",
+    "In open-loop evaluation we evaluate our model prediction as we follow the annotated ground truth.\n",
+    "\n",
+    "In each frame, we compare the predictions of our model against the ones annotated. This can be done with different metrics, such as ADE (Average Displacement Error) or FDE (Final Displacement Error). \n",
```

Comment:
When not knowing these metrics here, I'd be compelled to google.
Remove the names here, maybe simply spell 'measure distance to ref. trajectory'?